### ISSUE TITLE
Return execution resources of a test from `PureCairoTestCaseRunner`

### ISSUE THREAD

**Issue by Arcticae** [software-mansion/protostar#1285]
This might be tricky and some of the resources might be meaningless, because we'll use "starknet fixture" to execute contracts functions. 
The question is if we should assess the performance of the cairo code itself or include the starknet fixture resources used as well, or both with a separation of output (test code resources/SN fixture resources used respectively).

**Comment by Arcticae** [comment]
@MaksymilianDemitraszek would use an opinion of yours on this one

**Comment by Arcticae** [comment]
This is needed for profiling, putting this on hold

**Comment by Arcticae** [comment]
Blocked by https://github.com/software-mansion/protostar/milestone/31
